pom.xml

```diff
@@ -17,7 +17,7 @@
 
   <name>Drools and jBPM integration multiproject</name>
   <description>
-    Drools and jBPM integration with seam, spring, camel, ...
+    Drools and jBPM integration with seam, spring, camel, ....
```

Comment:
come on 4 dots instead of three ... that's unacceptable!! :)

Reply:
I am going to the shame corner right now...